db/migrate/20231102120748_add_notice_parents_for_account.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddNoticeParentsForAccount < ActiveRecord::Migration[6.1]
+  def change
+    add_column :accounts, :notice_parents, :boolean, default: true
```

Comment:
```suggestion
    add_column :accounts, :notify_parents, :boolean, default: true
```

notice has a bit different meaning - https://dictionary.cambridge.org/dictionary/english/notice